lib/rialto/etl/cli/error_reporter.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require 'honeybadger'
+
+module Rialto
+  module Etl
+    module CLI
+      # Logs errors to STDERR and to Honeybadger
+      class ErrorReporter
+        # rubocop:disable Style/StderrPuts
+        def self.log_exception(message)
```

Comment:
Instead of outputting the message to STDERR, should the CLI classes use the `Rialto::Etl::Logging` module so that the errors are logged to files that are shippable to S3? cc: @aaron-collier

Reply:
Discussed. 👍